advanced/Scripts/updatecheck.sh

```diff
@@ -44,19 +44,32 @@ if [[ "$2" == "remote" ]]; then
         sleep 30
     fi
 
-    GITHUB_CORE_VERSION="$(curl -s 'https://api.github.com/repos/pi-hole/pi-hole/releases/latest' 2> /dev/null | jq --raw-output .tag_name)"
+    # Source the COL_TABLE
+    # shellcheck disable=SC1091
+    . /opt/pihole/COL_TABLE
+
+    apidomain="api.github.com"
```

Comment:
Can I request that we remove this specific changeset for the time being - and address in a separate PR if necessary? 

Currently I'm of the opinion that checking if `api.github.com` is resolvable is an unnecessary addition to code complexity with very little gain

Reply:
removed...